.github/workflows/ci.yml

```diff
@@ -35,10 +35,12 @@ jobs:
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/previous-releases
         node-version: [18.x, 20.x]
-        suite: [commonjs, esm, typescript, cloudflare-worker]
+        suite: [commonjs, esm, typescript, cloudflare-worker, browser]
         exclude:
           - suite: cloudflare-worker
             node-version: 18.x # Only test Cloudflare suite with the latest Node version
+          - suite: browser
+            node-version: 18.x # Only test Cloudflare suite with the latest Node version
```

Comment:
I'll refactor this to match the Bun example in #220 once that lands

Reply:
Just landed that, so fire away!

Reply:
Done, assuming the CI passes :)